build.sh

```diff
@@ -23,6 +23,22 @@ usage()
     exit 1
 }
 
+initDistroName()
+{
+    if [ "$__BuildOS" == "Linux" ]; then
```

Comment:
Also the most portable way is:  `[ "x$var" = "xVALUE" ]` -- empty strings may behave differently in diverse shells.